### DISCUSSION TITLE
(Projects Beta) Add Projects Beta to Teams

### DISCUSSION THREAD

**Discussion by chriskuech** [community/community#9215] [Projects and Issues]
Currently Teams contains tab for Projects, but (unlike in organizations) only Projects is shown with no indication that Projects Beta exists.  This has been confusing when trying to onboard new team members.

Solutions would be any of
* adding Projects Beta support to Teams
* allow disabling Projects (original) from an organization entirely

I know this is on your roadmap, but prioritizing this would help a lot.

**Comment by trylaarsdam** [comment]
Definitely. Just created my first projects beta and tried to assign it to a team in an organization only to realize it isn't possible.

**Reply by stephan-uhlmann** [reply]
I'm missing that feature, too.

**Reply by LeninParedes** [reply]
I can't add beta projects to my team but I can add normal projects. Is it a design limitation or is it an issue?

**Reply by jlunder00** [reply]
https://github.com/github/roadmap/issues/563

**Comment by djkeh** [comment]
I have the same issue, too. Any concerns?

**Comment by agarciacalvo** [comment]
+1 here!

**Comment by jlursenbach** [comment]
This is a pretty big issue in my opinion! The Projects Beta is so much better for trying to scrum with my dev team, we really need this

**Comment by askpt** [comment]
Yeah! That's very confusing indeed! I spent a couple of minutes trying to figure out permissions and so on just to find in the end that is not working properly!

**Comment by Oskang09** [comment]
same issues, trying to work on it found that unable to add lolz

**Comment by nikitaksv** [comment]
Is someone working on this now?

**Reply by jlunder00** [reply]
It's on the roadmap: https://github.com/github/roadmap/issues/563

No idea when it will actually be shipped though...

**Comment by nikmart** [comment]
I was also not able to add a project to one of my teams. It seems that I cannot create a "classic" project either that would allow me to add the project to the team. It does seems like I can add my team to access the project. But it would be nice to have all the assets for my team under my team's page.

**Reply by bitfrost** [reply]
I 100% agree.  This not working is driving me crazy / driving me away from further adopting projects / using the teams feature

**Reply by jlunder00** [reply]
I am having the same exact issue. I'm able to create my teams and my projects in the organization, but I can't add the projects to the teams page, because it's searching for classic projects and not "beta" projects (is this still beta?). Why can't the teams page have the option to search for/add either???

Edit: our solution is on the roadmap, no idea if it will be shipped soon though: https://github.com/github/roadmap/issues/563

**Comment by jlunder00** [comment]
This issue really needs to be prioritized, since classic projects are being phased out, and you are not able to create classic projects unless the organization already has a classic project, new organizations, like the one I am working on, are completely unable to add project boards to a team. Currently, the teams project page is expecting classic projects only, and cannot find any of the modern projects boards in the organization regardless of access rights/permissions.

Edit: this is on the roadmap: https://github.com/github/roadmap/issues/563

**Comment by askpt** [comment]
It seems this was released a couple of hours ago! I can now add the new projects to the Team project view! https://github.com/github/roadmap/issues/563#issuecomment-1266135103
